src/index.html

```diff
@@ -2,11 +2,314 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="navigation">
+          <a
+            href="#home"
+            class="navigation__logo-link"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+              class="navigation__logo-image"
+            >
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#apple"
+                  class="nav__link is-active"
+                >Apple</a>
```

Comment:
please check the spaces everywhere. and the tag could be written in one line if there are less than 3 attributes
```suggestion
                <a href="#apple" class="nav__link is-active">
                  Apple
                </a>
```